clouddq/runners/dbt/dbt_runner.py

```diff
@@ -123,93 +117,93 @@ def get_entity_summary_path(self) -> Path:
+        )
+        self.connection_config = connection_config
+        self.dbt_profiles_dir = Path(self.dbt_path)
+        logger.debug(
+            "Using dbt profiles.yml path: {self.dbt_profiles_dir}",
+        )
+        if environment_target:
+            logger.debug(f"Using `environment_target`: {environment_target}")
             self.environment_target = environment_target
-        else:
-            # create GcpDbtConnectionConfig
-            connection_config = GcpDbtConnectionConfig(
-                gcp_project_id=gcp_project_id,
-                gcp_bq_dataset_id=gcp_bq_dataset_id,
-                bigquery_client=bigquery_client,
-                gcp_region_id=gcp_region_id,
-                gcp_service_account_key_path=gcp_service_account_key_path,
-                gcp_impersonation_credentials=gcp_impersonation_credentials,
+            self.connection_config.to_dbt_profiles_yml(
+                target_directory=self.dbt_profiles_dir,
+                environment_target=self.environment_target,
             )
-            self.connection_config = connection_config
-            self.dbt_profiles_dir = Path(self.dbt_path)
-            logger.debug(
-                "Using dbt profiles.yml path: {self.dbt_profiles_dir}",
+        else:
+            self.environment_target = DEFAULT_DBT_ENVIRONMENT_TARGET
+            self.connection_config.to_dbt_profiles_yml(
+                target_directory=self.dbt_profiles_dir
             )
-            if environment_target:
-                logger.debug(f"Using `environment_target`: {environment_target}")
-                self.environment_target = environment_target
-                self.connection_config.to_dbt_profiles_yml(
-                    target_directory=self.dbt_profiles_dir,
-                    environment_target=self.environment_target,
-                )
-            else:
-                self.environment_target = DEFAULT_DBT_ENVIRONMENT_TARGET
-                self.connection_config.to_dbt_profiles_yml(
-                    target_directory=self.dbt_profiles_dir
-                )
 
     def _resolve_dbt_path(
         self,
-        dbt_path: str,
         create_paths_if_not_exists: bool = False,
         write_log: bool = False,
     ) -> Path:
         logger.debug(f"Current working directory: {Path().cwd()}")
-        if not dbt_path:
-            dbt_path = Path().cwd().joinpath("dbt").absolute()
-            logger.debug(
-                "No argument 'dbt_path' provided. Defaulting to use "
-                f"'dbt' directory in current working directory at: {dbt_path}"
-            )
-        else:
-            dbt_path = Path(dbt_path).absolute()
-            if dbt_path.name != "dbt":
-                dbt_path = dbt_path / "dbt"
+        dbt_path = Path().cwd().joinpath("dbt").absolute()
+        logger.debug(
+            "No argument 'dbt_path' provided. Defaulting to use "
+            f"'dbt' directory in current working directory at: {dbt_path}"
+        )
```

Comment:
Should we update the logging message?

Reply:
updated to: **Defaulting to use 'dbt' directory in current working directory at: {dbt_path}**